### ISSUE TITLE
Failed To Parse Export

### ISSUE THREAD

**Issue by Kaiverrus** [atenfyr/UAssetGUI#70]
Keep getting "Failed to parse 1 exports!" when trying to open any files from this game
Game: F1 Manager 2023
Engine: I believe run on 5.1 since using 5.1 below won't work at all

I attach few files to be tested
[F1M23.zip](https://github.com/atenfyr/UAssetGUI/files/12196560/F1M23.zip)

**Comment by fibonatee** [comment]
Second that. I also had this problem when opening uasset files. Let me know how it works out for you, thx.

**Comment by LongerWarrior** [comment]
Game is using Unversioned properties, so u need to generate mappings and  then import then in UAssetGUI

**Comment by Kaiverrus** [comment]
Tried that but I got error from UAssetGUI.
"Index was out of range. Must be non-negative and less than the size of collection. Parameter name: Index."

Attached mappings
[Mappings.zip](https://github.com/atenfyr/UAssetGUI/files/12207273/Mappings.zip)

**Comment by Kaiverrus** [comment]
Okay just learned that the mapping issue is from dumper itself. I guess gotta wait till the dumper get fixed

**Comment by LongerWarrior** [comment]
yes that is the problem with UnrealMappingsDumper generate incorrect mapping for big enums
here is valid mappings
[F1Manager2023.zip](https://github.com/atenfyr/UAssetGUI/files/12207426/F1Manager2023.zip)

**Comment by Kaiverrus** [comment]
Thanks! No more mapping error now. Just the "Failed to parse export" still exist

**Comment by Kaiverrus** [comment]
@atenfyr Sorry for the mention, is there any ETA on when this get fixed?

**Comment by fibonatee** [comment]
> yes that is the problem with UnrealMappingsDumper generate incorrect mapping for big enums here is valid mappings [F1Manager2023.zip](https://github.com/atenfyr/UAssetGUI/files/12207426/F1Manager2023.zip)

Thx, would be awesome if we can get an update for Game: F1 Manager 2023. Right now it fails to parse 1 export, failed to maintain binary equality.
![Snap 2023-08-02 at 12 03 24](https://github.com/atenfyr/UAssetGUI/assets/140854594/ace775a9-b23b-487b-8b5b-93e4f045e942)

**Comment by fibonatee** [comment]
Good day, any progress guys？
